### PR TITLE
Update/remove internal broken links

### PR DESCRIPTION
**Pull request by OnkarRuikar**
No alternates have been provided for outdated XUL APIs.

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/3.5/index.md [hunk 1]
```diff
@@ -91,39 +91,39 @@ tags:
...
   - : Firefox 3.5 supports the Geolocation API, which allows web applications to obtain information about the user's current location if a provider for that information is installed and enabled.
 - [Locating DOM elements using selectors](/en-US/docs/Web/API/Document_object_model/Locating_DOM_elements_using_selectors)
   - : The selectors API allows querying a document to locate the elements that match a given selection rule.
-- [Mouse gesture events](/en-US/docs/Web/Events/Mouse_gesture_events)
+- [Mouse gesture events](https://web.archive.org/web/20210724051233/https://developer.mozilla.org/en-US/docs/Web/Events/Mouse_gesture_events)
   - : Firefox 3.5 supports mouse gesture events such as trackpad swipes.
 - [The `NodeIterator` object](/en-US/docs/Web/API/NodeIterator)
   - : The `NodeIterator` object provides support for iterating over the list of the nodes in a DOM subtree.
-- [The MozAfterPaint event](/en-US/docs/Gecko-Specific_DOM_Events#mozafterpaint)
+- The MozAfterPaint event
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/3.5/index.md:102]:
```suggestion
- The `MozAfterPaint` event
```

File: files/en-us/mozilla/firefox/releases/3.5/index.md [hunk 2]
```diff
@@ -91,39 +91,39 @@ tags:
...
   - : This new DOM event is sent after painting updates in windows.
-- [The MozMousePixelScroll event](/en-US/docs/Gecko-Specific_DOM_Events#mozmousepixelscroll)
+- The MozMousePixelScroll event
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/3.5/index.md:104]:
```suggestion
- The `MozMousePixelScroll` event
```

File: files/en-us/mozilla/firefox/releases/3.5/index.md [hunk 3]
```diff
@@ -91,39 +91,39 @@ tags:
...
   - : This new DOM event allows detection of pixel-based mouse scroll wheel events instead of line-based scroll events.
 
 #### New JavaScript features
 
-- [New in JavaScript 1.8.1](/en-US/docs/JavaScript/New_in_JavaScript/1.8.1)
+- [New in JavaScript 1.8.1](https://developer.mozilla.org/en-US/docs/Archive/Web/JavaScript/New_in_JavaScript/1.8.1)
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/3.5/index.md:109]: This link doesn't work anymore

File: files/en-us/mozilla/firefox/releases/3.5/index.md [hunk 4]
```diff
@@ -139,12 +139,12 @@ tags:
 
 #### New SVG features
 
-- [Applying SVG effects to HTML content](/en-US/Applying_SVG_effects_to_HTML_content)
+- [Applying SVG effects to HTML content](/en-US/docs/web/svg/applying_svg_effects_to_html_content)
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/3.5/index.md:142]: Capitalization of the slug is not correct
```suggestion
- [Applying SVG effects to HTML content](/en-US/docs/Web/SVG/Applying_SVG_effects_to_HTML_content)
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/mozilla/firefox/releases/3.5/index.md [hunk 5]
```diff
@@ -91,39 +91,39 @@ tags:
...
+- The `MozAfterPaint` event
   - : This new DOM event is sent after painting updates in windows.
-- [The MozMousePixelScroll event](/en-US/docs/Gecko-Specific_DOM_Events#mozmousepixelscroll)
+- The `MozMousePixelScroll` event
   - : This new DOM event allows detection of pixel-based mouse scroll wheel events instead of line-based scroll events.
 
 #### New JavaScript features
 
-- [New in JavaScript 1.8.1](/en-US/docs/JavaScript/New_in_JavaScript/1.8.1)
+- [New in JavaScript 1.8.1](https://developer.mozilla.org/en-US/docs/Archive/Web/JavaScript/New_in_JavaScript/1.8.1)
```
Comment (OnkarRuikar) [inline files/en-us/mozilla/firefox/releases/3.5/index.md:109]: forgot to copy the web archive link 
```suggestion
- [New in JavaScript 1.8.1](https://web.archive.org/web/20200805002842/https://developer.mozilla.org/en-US/docs/Archive/Web/JavaScript/New_in_JavaScript/1.8.1)
```

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**